testdata/ansible/memcached-operator/Makefile

```diff
@@ -113,3 +120,42 @@ bundle: kustomize
 .PHONY: bundle-build
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+
+# Push the bundle image.
+.PHONY: bundle-push
+bundle-push:
+	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
+
+# Download opm locally if necessary.
+.PHONY: opm
+OPM = ./bin/opm
+opm:
+ifeq (,$(wildcard $(OPM)))
+ifeq (,$(shell which opm 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPM)) ;\
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$(OS)-$(ARCH)-opm ;\
+	chmod +x $(OPM) ;\
+	}
+else
+OPM = $(shell which opm)
+endif
+endif
+
+# A comma-separated list of bundle images (e.g. make catalog-build BUNDLE_IMGS=example.com/operator-bundle:v0.1.0,example.com/operator-bundle:v0.2.0).
+# These images MUST exist in a registry and be pull-able.
+BUNDLE_IMGS ?= $(BUNDLE_IMG)
+# The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
+CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:$(VERSION)
+# Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
```

Comment:
Why/when/who requires a catalogue image with many bundles of the same project?
How users can benefit from this and in what scenario? IMO we need to add a doc as well describing that.

Reply:
Everyone who uses catalogs. The whole point of a catalog is so multiple versions of an operator can be chosen from in a convenient manner.